### PR TITLE
[Feature]Adding toggle button for advanced filter and updating css

### PR DESCRIPTION
**Pull request by Jeevan1**

### CODE REVIEW

File: src/components/LoanFilterForm.jsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const OptionWrapper = styled.div`
   }
 `;
 
-function LoanFilterForm() {
+function LoanFilterForm({ show }) {
```
Comment (RahulRimal) [inline src/components/LoanFilterForm.jsx:91]: change prop to showFilters

File: src/components/LoanFilterForm.jsx [hunk 2]
```diff
@@ -178,7 +178,7 @@ function LoanFilterForm() {
   ];
 
   return (
-    <div>
+    <div className={show ? "show" : "hide"}>
```
Comment (RahulRimal) [inline src/components/LoanFilterForm.jsx:181]: change to showFilters

File: src/components/Tables/BaseTable.jsx [hunk 3]
```diff
@@ -160,7 +161,7 @@ const BaseTable = ({
   navigateOnRowClick,
 }) => {
   const theme = useTheme();
-
+  const [showFilterForm, setShowFilterForm] = React.useState(false);
```
Comment (RahulRimal) [inline src/components/Tables/BaseTable.jsx:164]: better to import useState above and use useState here

File: src/components/Tables/BaseTable.jsx [hunk 4]
```diff
@@ -160,7 +161,7 @@ const BaseTable = ({
...
   const [sorting, setSorting] = React.useState([]);
   const [filtering, setFiltering] = React.useState("");
```
Comment (RahulRimal) [inline src/components/Tables/BaseTable.jsx:166]: better to import useState above and use useState here

File: src/components/Tables/BaseTable.jsx [hunk 5]
```diff
@@ -160,7 +161,7 @@ const BaseTable = ({
   navigateOnRowClick,
 }) => {
   const theme = useTheme();
-
+  const [showFilterForm, setShowFilterForm] = React.useState(false);
   const [sorting, setSorting] = React.useState([]);
```
Comment (RahulRimal) [inline src/components/Tables/BaseTable.jsx:165]: better to import useState above and use useState here

**Review by RahulRimal — CHANGES_REQUESTED** [review]

File: src/components/LoanFilterForm.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/components/LoanFilterForm.jsx:91]: done

File: src/components/LoanFilterForm.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/components/LoanFilterForm.jsx:181]: done

File: src/components/Tables/BaseTable.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/components/Tables/BaseTable.jsx:164]: done

File: src/components/Tables/BaseTable.jsx (hunk 4 above)
Comment (Jeevan1) [reply src/components/Tables/BaseTable.jsx:166]: done

File: src/components/Tables/BaseTable.jsx (hunk 5 above)
Comment (Jeevan1) [reply src/components/Tables/BaseTable.jsx:165]: done

**Pull request merged**